cmake/modules/CheckCompiler.cmake

```diff
@@ -192,8 +193,19 @@ check_cxx_source_compiles(
 int main() {}
 " GLIBCXX_USE_CXX11_ABI)
 
+
```

Comment:
Why add the extra blank line?

Reply:
Looks cleaner in my eyes, but I don't insist.